manifests/istio-operator/templates/clusterrole.yaml

```diff
@@ -0,0 +1,113 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
why is this added?

Reply:
In the original PR it is a move, but for 1.5 I'm keeping the original templates in case someone relies on the old path, which is why this shows up as an add, not move.

Reply:
Why is this needed even in the original pr? how does the operator control impact local install (istioctl)? is it for `istioctl operator init`?